sw/device/lib/dif/dif_plic.c

```diff
@@ -53,13 +53,13 @@ typedef struct plic_target_reg_offset {
 // target, so there should be `RV_PLIC_PARAM_NUMTARGET` entries in this array.
 // The `i`th entry should contain the offsets of the `i`th target specific
 // registers:
-// - `RV_PLIC_IE<i>0_REG_OFFSET` (the first IE reg for target `i`).
+// - `RV_PLIC_IE<i>_0_REG_OFFSET` (the first IE reg for target `i`).
 // - `RV_PLIC_CC<i>_REG_OFFSET`
 // - `RV_PLIC_THRESHOLD<i>_REG_OFFSET`
 static const plic_target_reg_offset_t plic_target_reg_offsets[] = {
         [0] =
             {
-                .ie = RV_PLIC_IE00_REG_OFFSET,
+                .ie = RV_PLIC_IE0_0_REG_OFFSET,
```

Comment:
IIUC `IE0` here is part of the base name and this is expected, right?

Reply:
Yep, that's correct, `IE0` is the base name, it stands for Target 0. I think the number of targets is not fully parametrized at the moment. This would require multi-multiregs (2D), in which case the name would probably become `IE_0_0`.

Thanks for your review @alphan .